jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml

```diff
@@ -162,6 +162,7 @@
                 # So setting max-nodes-per-pool=1000, to check if that helps.
                 export GKE_CREATE_FLAGS="--max-nodes-per-pool=1000"
                 export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=False
+								export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://staging-container.sandbox.googleapis.com/"
```

Comment:
Nit: the tabs here are way off. (It might be a more than nit if it violates the YAML)